stack/docker-compose.yml

```diff
@@ -107,14 +107,16 @@ services:
     - SSO_CLIENT_ID=local
     - SSO_CLIENT_SECRET=client_secret
     - SPRING_CLOUD_GATEWAY_ROUTES_0_ID=backend
-    - SPRING_CLOUD_GATEWAY_ROUTES_0_URI=http://backend-service:39146/
+    - SPRING_CLOUD_GATEWAY_ROUTES_0_URI=http://host.docker.internal:39146/
```

Comment:
Why is this needed? Can't the API-Gateway talk to the backend because there are not in the same Docker network and thus, this special DNS name by docker is necessary?

Reply:
Yes